src/net/java/sip/communicator/plugin/ircaccregwizz/IrcIgnoreConfigForm.java

```diff
@@ -0,0 +1,215 @@
+package net.java.sip.communicator.plugin.ircaccregwizz;
+
+import net.java.sip.communicator.impl.protocol.irc.properties.*;
```

Comment:
You'll need to copy the property. The ircaccregwiz is a different bundle and cannot use anthing from the protocol impl.